src/olympia/addons/migrations/0051_remove_accessibility.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 4.2.13 on 2024-05-22 11:39
+
+from django.db import migrations, models
+from olympia import amo
+from olympia.constants.categories import CATEGORIES
+
+
+def remove_accessibility_category(apps, schema_editor):
```

Comment:
As a developer, when you select the `Other` category you can't choose another category - if you choose that one it should be the only one for your add-on. If we did a replace here, potentially some add-ons would go against that and end up with `Other` and up to 2 other categories, and those add-ons would be a weird state (next time they try to edit metadata they would get an error and need to fix that)

Now, in some instances when doing category migrations we haven't been able to avoid that, but here we can so I figured I would.